lib/mapper.js

```diff
@@ -10,19 +10,50 @@ var unixTime = require('unix-time');
 var del = require('obj-case').del;
 var reject = require('reject');
 
+function copyFields(fromObj, toObj, fields) {
```

Comment:
I'd prefer if we used https://github.com/ndhoule/pick and https://github.com/ndhoule/extend but it's not a deal breaker.